### DISCUSSION TITLE
Actividad 0 - Rodriguez, Luciano

### DISCUSSION THREAD

**Discussion by LucianooRodriguez** [frlp-utn-ingsoft/catedra#66] [Show and tell]
https://github.com/LucianooRodriguez/IngenieriaSoftware

**Comment by RodrigoJacznik** [comment]
Hola,
los mensajes de los commits podrian ser mejor (estan desordenadas las partes y algunos commits tienen los mensajes repetidos)

La app tiene 2 errores:

1. En el index de polls (el nombre del archivo no se debería ver). 
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/2715743/25c1feec-0a26-4817-95d5-b43411b69a3c)
2. Al entrar en una `poll` no hay `choice` cargadas. Una `poll` se debería ver mas o menos así:
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/2715743/6198e118-075c-4cb2-a737-ad2278f453ee)
Creo que te falto la ultima parte de esta sección https://docs.djangoproject.com/en/5.0/intro/tutorial02/#playing-with-the-api, la que esta justo antes de "Introducing the Django Admin".
En la parte 7 (https://docs.djangoproject.com/en/5.0/intro/tutorial07/) se muestra como modificar el admin para agregar `choice` a una `poll`. Esta parte no es obligatorio que lo hagas pero quizás es mas fácil que agregar las `choice` mediante la shell de django.

Arregla estos 2 puntos para aprobar la actividad

**Reply by LucianooRodriguez** [reply]
Buenos días, ya arreglé la actividad. Cualquier duda a disposición. Saludos.

**Reply by RodrigoJacznik** [reply]
Ahora si 👏.
La actividad esta aprobada.
